src/Examples/UnitModels/cstr_testing.ipynb

```diff
@@ -0,0 +1,1107 @@
+    "\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"ethylene_oxide\"].fix(1e-5*pyunits.mol/pyunits.s)\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"water\"].fix(200*pyunits.mol/pyunits.s)\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"sulfuric_acid\"].fix(0.334*pyunits.mol/pyunits.s)  # calculated from 0.9 wt% SA in stream\n",
+    "m.fs.M101.catalyst_feed.flow_mol_phase_comp[0, \"Liq\", \"ethylene_glycol\"].fix(1e-5*pyunits.mol/pyunits.s)\n",
+    "m.fs.M101.catalyst_feed.temperature.fix(298.15*pyunits.K)\n",
+    "m.fs.M101.catalyst_feed.pressure.fix(1e5*pyunits.Pa)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fixing unit model specifications\n",
+    "\n",
+    "Now that we have fixed our inlet feed conditions, we will now be fixing the operating conditions for the unit models in the flowsheet. Let us initialize H101 with an outlet temperature of 328.15 K. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.fs.H101.outlet.temperature.fix(328.15*pyunits.K)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll add constraints defining the reactor volume and conversion in relation to the stream properties. Particularly, we want to use our CSTR performance relation:  \n",
+    "\n",
+    "$V = \\frac{v_0 X} {k(1-X)}$, where the CSTR reaction volume $V$ will be specified, the inlet volumetric flow $v_0$ is determined by stream properties, $k$ is calculated by the reaction package, and $X$ will be calculated. Reactor volume is commonly selected as a specification in simulation problems, and choosing conversion is often to perform reactor design.\n",
+    "\n",
+    "Typically in reactor sizing, the liquid volume (reaction volume) does not exactly reach the top of the reactor vessel due to overhead space, vapor draws, or pressure-release valves. We will assume that 80% of our total reactor volume (vessel size, normally used for equipment costing) is our liquid, which is a common feature of stirred-tank or batch reactors. For the CSTR, we have to define the conversion in terms of ethylene oxide as well as the CSTR reaction volume. This requires us to create new variables and constraints relating reactor properties to stream properties.  Note that the CSTR reaction volume variable (m.fs.R101.volume) does not need to be defined here since it is internally defined by the CSTR model. Additionally, the heat duty is not fixed, since the heat of reaction depends on the reactor conversion (through the extent of reaction and heat of reaction). We'll estimate 80% conversion for our initial flowsheet:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m.fs.R101.total_volume = Var(initialize=6.9225, bounds=(0, 15), units=pyunits.m**3)  # m^3\n",
```

Comment:
Minor comment: This new var (`total_volume`) could have been avoided as we did not include the capex anywhere in the optimization. Even if we included the capex, we could have used the reactor volume var and multiplied it by a factor of 1.2 to give the 20% design margin.

Reply:
Would you suggest removing this variable? I believed it is an important point to mention regarding reactor design, and I agree a design margin may be less confusing to users than an additional variable.

Reply:
I would @bpaul4. A comment will be sufficient or you can print the total volume to be 1.2 times the reactor volume you get.